src/scripts/core/models/grid.ts

```diff
@@ -8,22 +8,18 @@ export class Grid extends Container {
     private gridSize: number;
     private columns: Column[];
     private activeToken: number;
-
+    private firstTokenLocation: number[];
```

Comment:
Is this intended to always represent an x and y co-ordinate in your grid?

If so it could be typed better than just `number[]` which can be an array of any length.

- You could make it a tuple (`private location: [number, number]`).
This is still an array and will work with what you currently have, except TypeScript will prevent you from accidentally assigning/reading anything other than the first 2.

- You could also use the type `IPointData` which Pixi provides and is just a basic `{ x: number; y: number }` structure.